flight/targets/quanton/board-info/board_hw_defs.c

```diff
@@ -1816,6 +1816,60 @@ static const struct pios_tim_channel pios_tim_servoport_rcvrport_pins[] = {
 };
 
 #if defined(PIOS_INCLUDE_SERVO) && defined(PIOS_INCLUDE_TIM)
+
+#if defined(PIOS_INCLUDE_DMASHOT)
+
+#include <pios_dmashot.h>
+
+/*
+TIM8 C4, C3
+TIM14 C1
+TIM3 C1, C3, C4
+TIM5 C1, C2
+*/
+
+static const struct pios_dmashot_timer_cfg dmashot_tim_cfg[] = {
+	{
+		.timer = TIM3,
+		.stream = DMA1_Stream2,
+		.channel = DMA_Channel_5,
+		.tcif = DMA_FLAG_TCIF2,
+	},
+	{
+		.timer = TIM1,
+		.stream = DMA2_Stream5,
+		.channel = DMA_Channel_6,
+		.tcif = DMA_FLAG_TCIF5,
+		// Shares DMA channel with WS2811!
+		// DMAShot requests very high priority, so should get preference.
```

Comment:
Channels are per-stream.. I don't see any use of DMA2_Stream5 elsewhere.

Priority is just used to select which of multiple streams that want to DMA at the same time... who wins.